### ISSUE TITLE
`expect` example from README doesn't work

### ISSUE THREAD

**Issue by LilithHafner** [JuliaLang/juliaup#464]
I copied and pasted the script installation code from README.md and it did not work for me.

```
Last login: Thu Nov 24 08:03:02 on ttys006
x@X ~ % expect -c 'spawn bash -c "curl -fsSL https://install.julialang.org | sh"; expect "Proceed"; send -- "\r"; expect eof'
spawn bash -c curl -fsSL https://install.julialang.org | sh

info: downloading installer
Welcome to Julia!

This will download and install the official Julia Language distribution
and its version manager Juliaup.

Juliaup will be installed into the Juliaup home directory, located at:

  /Users/x/.julia/juliaup

The julia, juliaup and other commands will be added to
Juliaup's bin directory, located at:

  /Users/x/.julia/juliaup/bin

This path will then be added to your PATH environment variable by
modifying the profile files located at:

  /Users/x/.profile
  /Users/x/.bash_profile
  /Users/x/.zshrc

Julia will look for a new version of Juliaup itself every 1440 minutes when you start julia.

You can uninstall at any time with juliaup self uninstall and these
changes will be reverted.

✔ Do you want to install with these default configuration choices? · Proceed with installation

Now installing Juliaup
Installing Julia 1.8.3+0.x64 (x64).
  Downloading: [==>                 ] 15.02 MiB/102.54 MiB eta: 62s
x@X ~ % ~/.julia/juliaup/bin/julia 
zsh: no such file or directory: /Users/x/.julia/juliaup/bin/julia
x@X ~ % ~/.julia/juliaup/bin/julialauncher 
Error: The Julia launcher failed to figure out which juliaup channel to use.
x@X ~ % . /Users/x/.zshrc
x@X ~ % julia                             
zsh: command not found: julia
x@X ~ % ls ~/.julia/juliaup 
bin			juliaup.json
julia-1.8.3+0.x64	juliaupself.json
x@X ~ % ls ~/.julia/juliaup/bin 
julialauncher	juliaup
x@X ~ % juliaup status
 Default  Channel  Version  Update 
-----------------------------------
x@X ~ % 
```
and in a new shell
```
Last login: Thu Nov 24 08:03:14 on ttys006
x@X ~ % julia
zsh: command not found: julia
x@X ~ % ls ~/.julia/juliaup/bin
julialauncher	juliaup
x@X ~ % 
```
Here's my platform info courtesy of `versioninfo()`
```
Platform Info:
  OS: macOS (x86_64-apple-darwin21.4.0)
  CPU: 4 × Intel(R) Core(TM) i5-8210Y CPU @ 1.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, skylake)
  Threads: 1 on 2 virtual cores
```

**Comment by babaq** [comment]
I have a related issue, when i tried to use juliaup to setup julia installation in azure pipelines, ubuntu vm Microsoft provided doesn't have `expect` tools, so i am wandering if the install script can add a option to silently install juliaup using the defaults.

**Comment by LilithHafner** [comment]
Yes. That would be the best solution here. See #218 (tagged with help wanted, so PRs would be very welcome)

**Comment by LilithHafner** [comment]
Fixed by #548
